### DISCUSSION TITLE
Can't locate database connection string

### DISCUSSION THREAD

**Discussion by mrpavlitzky** [OrchardCMS/OrchardCore#12143] [Q&A]
Hi,

I just migrated from running Orchard from Nuget package to the full source code drop and I have a following 'issue'.
I can't locate the db connection string! I stored it in the Azure environment variables and just to see what's the actual source of it, I changed it to a wrong one, saved, reloaded the page and... the site still loads the contents properly. The connection string is not present in appsettings.json and the folder App_Data is missing from the solution. Any ideas where it might be stored?

**Comment by Skrypt** [comment]
Azure blob storage?

**Reply by mrpavlitzky** [reply]
Nope, it's never been configured to store connection string. I used only Azure environment variables.

**Comment by ns8482e** [comment]
I guess connection string is stored under `App_Data\Sites\Default\appsettings.json`

**Reply by mrpavlitzky** [reply]
Yes that used to be the case when I was running Orchard from the Nuget package, but this directory is absent now in the source code drop.

**Comment by hyzx86** [comment]
Have you specified such a configuration on the server `ORCHARD_APP_DATA`
`Environment=ORCHARD_APP_DATA=/var/www/JZSoft/easyoc/App_Data`

**Reply by mrpavlitzky** [reply]
Well, it kind of made it way back to my directory. I have no clue whatsoever what happened, but the next day the AppData was back in its place!

**Reply by hyzx86** [reply]
🤣
